addons/stock/models/stock_picking.py

```diff
@@ -1358,6 +1361,20 @@ def action_split_transfer(self):
         backorder_moves = moves._create_backorder()
         backorder_moves += self.move_ids.filtered(lambda m: m.quantity == 0)
         self._create_backorder(backorder_moves=backorder_moves)
+        action = self.env.ref('stock.stock_picking_action_picking_type')
+        return {
+            'type': 'ir.actions.client',
+            'tag': 'display_notification',
+            'params': {
+                'message': 'New transfer created %s',
```

Comment:
Move the text to `title` instead, only display the link in the `message`. It also needs to be translatable.